src/index.html

```diff
@@ -11,7 +11,146 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-set">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          required
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
            required
          >
```

![image](https://user-images.githubusercontent.com/80700247/231739771-8ff72d13-73fc-4042-b736-dd3861fe14e5.png)
![image](https://user-images.githubusercontent.com/80700247/231739815-9c779e1d-72a9-4a7f-97cb-939c5f578830.png)

Fix All Such Cases